decidim-participatory_processes/app/forms/decidim/participatory_processes/admin/participatory_process_import_form.rb

```diff
@@ -33,8 +33,7 @@ class ParticipatoryProcessImportForm < Form
         attribute :import_components, Boolean, default: true
         attribute :document
 
-        validates :document, presence: true
-
+        validates :document, file_content_type: { allow: ACCEPTED_TYPES.values }, if: ->(form) { form.title.blank? }
```

Comment:
As far as I understand this, you are checking if the title is blank because it will always be blank during the dynamic validation request but we assume the title is there when we **actually** submit the form.

But do we need the `if: ...` check here for anything? I tested without it and I couldn't submit the form if the file type was incorrect. It feels a bit hacky to me but there may be something here I don't understand.

Reply:
We discussed about this offline with @lahdeero and the problem here is that when the file is valid type (i.e. a JSON), the form shows a validation error because it is trying to validate a content type of a string (= the blob reference sent through the form).

To fix this we discussed about adding a new custom attribute type for these, so that custom type would be responsible of converting the string to a blob when the validation would work (as far as I understood from the conversation).

This would be similar e.g. to the `LocalizedDate` type which converts a Date object from a string.